website/views/shared/header/menu.jade

```diff
@@ -58,6 +58,8 @@ nav.toolbar(ng-controller='MenuCtrl')
               li
                 a(target='_blank' href='https://habitica.com/static/faq/')=env.t('FAQ')
               li
+                a(target='_blank' href='http://habitica.wikia.com/wiki/Habitica_Wiki')=env.t('HabiticaWiki')
```

Comment:
Links to any wiki page can be placed in the locales files so that translators can specify non-English wiki links where available. For example:

common/locales/en/tasks.json line 111:    "toDoHelp4": "For inspiration, check out these <a href='http://habitica.wikia.com/wiki/Sample_To-Dos' target='_blank'>sample To-Dos</a>!",